src/hats/pixel_math/hipscat_id.py

```diff
@@ -22,10 +22,10 @@
 
 import numpy as np
 
-import hipscat.pixel_math.healpix_shim as hp
+import hats.pixel_math.healpix_shim as hp
 
-HIPSCAT_ID_COLUMN = "_hipscat_index"
-HIPSCAT_ID_HEALPIX_ORDER = 19
+SPATIAL_INDEX_COLUMN = "_healpix_29"
+SPATIAL_INDEX_ORDER = 19
```

Comment:
I'm guessing this will become 29 in a later change when the mechanism for removing the uniqueness goes in. Correct?

Reply:
Yes - that's the change that's going to be more complicated!